src/renderer/src/App.css

```
@@ -454,6 +454,10 @@ a {
   break-after: always;
 }
 
+@media print {
+
+}
```

Stylelint (17.2.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)